src/main/tut/index.html

```diff
@@ -205,6 +237,7 @@
 
 ## Enhancement without inheritance!
 
+<!-- COMMENT -->
 Important realisation: _we've added a new feature to `Int`, even though it's `final`_.
```

Comment:
This is tricky, as I completely get were you're coming from here. In one sense, we have added that new functionality, but the intuition  with typeclasses is more - IMHO - about the encoding of behavior. When im teaching new folks about typeclasses in scala, I usually use the following intuition: 

In object orientation, Cow extends Animal extends Object (hierarchically speaking). Therefore we can say, `Cow` IS A `Animal`. Contrasted with typeclasses, we would encode the behavior of a cow, for example `Moo` and `Walk`, so the intuition is more HAS A... the `Cow` definition HAS A `Walk` and `Moo` typeclass, therefore it has those behaviors.